lib/ueberauth/strategy/application.ex

```diff
@@ -4,14 +4,20 @@ defmodule Ueberauth.Strategy.Passwordless.Application do
   alias Ueberauth.Strategy.Passwordless
 
   def start(_type, opts) do
-    if Passwordless.config(:use_store), do: start_store(opts)
+    :use_store
+    |> Passwordless.config()
+    |> maybe_start_store(opts)
```

Comment:
I don't see a difference between the old and new code. The old code also should not execute `start_store` if `:use_store` is set to false or not set at all. How is the new code different?

Reply:
Is the `{:ok, self()}` doing something special?

Reply:
The difference here is that there is a valid return value in the form of `{:ok, _pid}` if `:use_store` is set to false.

Launch the `:observer` in an app that uses the master branch of `UeberauthPasswordless` and see the started process of `UeberauthPasswordless`. You'll see that the `Store` is started by default. Now do the same but instead of the master branch use the `bugfix` one and you'll see that the `Store` is not started.

Reply:
I provided the `{:ok, self()}` because this is the only valid return value for when starting an application, otherwise an error is triggered.

See the [on_start](https://hexdocs.pm/elixir/Supervisor.html#t:on_start/0) return value which is a two element tuple with `:ok` and a `pid`.